src/librustc/hir/mod.rs

```diff
@@ -434,12 +434,12 @@ impl GenericArg {
 
 #[derive(RustcEncodable, RustcDecodable, Debug, HashStable)]
 pub struct GenericArgs {
-    /// The generic arguments for this path segment.
+    /// The generic parameters for this path segment.
```

Comment:
This is specifically arguments, not parameters.

Reply:
Both the issue (#62426) and the comment that introduced it (https://github.com/rust-lang/rust/pull/61856#discussion_r300805622) do not specify a formal set of rules on where an `argument`  or `parameter` should be used.
My understanding was that everything should be converted from `arg(s)` ->  `param(s)`. If not, a consensus should be reached to avoid yet another renaming in the future.

Reply:
>the comment that introduced it (#61856 (comment)) do not specify a formal set of rules on where an argument or parameter should be used.

Definition site - "parameter", use site - "argument".

Reply:
Changed